### ISSUE TITLE
RpiPowerController Button

### ISSUE THREAD

**Issue by s200bym** [vpanarello/RpiPowerController#1]
Hi, Can this circuit be used with a momentary push-button instead of a toggle switch?

Thanks.

**Comment by vpanarello** [comment]
Hi,
There are on file "Interface.cpp" 2 callbacks. One is called wenn switch turns on, other wenn turns off. I planned it first for switch, but, if you change the logic inside both functions, you can adjust it for push-buttons,

void turnedOn() {}
void turnedOff() {}

This both functions controls what will happen to Raspberry +Led.

**Comment by vrtlspd** [comment]
Hi, I want to make the same change, however I'm having a bad time changing the code. I'm not very familiar with c++ language. Can you help make the change to a momentary push button instead of a regular switch?

thank you.

**Comment by vpanarello** [comment]
Hi, Do you mean a push-button ?

I will try later to implement this logic and commit everything in a separeted branch. Unfortunelly I do not have the prepared hardware for testing it anymore.

> Hi, I want to make the same change, however I'm having a bad time changing the code. I'm not very familiar with c++ language. Can you help make the change to a momentary push button instead of a regular switch?
> 
> thank you.

**Comment by vrtlspd** [comment]
Yes, a momentary push button. I can test it and maybe even change the code if necessary.

Thank you
